src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.cs

```diff
@@ -17,20 +17,22 @@ internal abstract partial class AbstractFindUsagesService : IFindUsagesService
                 document, position, context.CancellationToken).ConfigureAwait(false);
             if (tuple == null)
             {
-                context.ReportMessage(EditorFeaturesResources.Cannot_navigate_to_the_symbol_under_the_caret);
+                await context.ReportMessageAsync(
+                    EditorFeaturesResources.Cannot_navigate_to_the_symbol_under_the_caret).ConfigureAwait(false);
                 return;
             }
 
             var message = tuple.Value.message;
 
             if (message != null)
             {
-                context.ReportMessage(message);
+                await context.ReportMessageAsync(message).ConfigureAwait(false);
                 return;
             }
 
-            context.SetSearchTitle(string.Format(EditorFeaturesResources._0_implementations,
-                FindUsagesHelpers.GetDisplayName(tuple.Value.symbol)));
+            await context.SetSearchTitleAsync(
```

Comment:
:memo: I commented because of this statement in the original request:

> ... and have it be called without the OOP process blocking on these calls as we chat ...

Which led me to the following specific observations:

1. The operation is just as "chatty" now as it was before.
2. The communication continues to block _progress_ in the same manner it did before.

During my work on the Java debugger for Visual Studio, pipelining OOP commands was one of the primary reasons for the substantial performance improvements I was able to achieve relative to debuggers which already existed.